### PR TITLE
New loader component

### PR DESCRIPTION
**Pull request by gmzjuliana**
## Description
Adds a loader component

## Motivation and Context
[Figma file](https://www.figma.com/file/onxcG6l5KavWZ4Z6gt8k9M/Loader-Component-Animation?node-id=0%3A1)

## Screenshots (if appropriate):
https://share.getcloudapp.com/X6uYPEAQ

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style and guidelines of this project. 
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the **CHANGELOG** document.
- [x] I have added tests to cover my changes.
- [x] I have performed a self-review of my own code
- [x] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] All new and existing tests passed.
- [x] I have performed the accessibility audit of my UI changes according to the accessibility doc. 
- [ ] [Buffer Engineers] Someone from the Design team reviewed and approved my changes
- [ ] [Buffer Engineers] I have notified the BDS team of my changes in the #proj-design-system Slack channel

### CODE REVIEW

File: src/documentation/examples/Loader/Loader.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Loader from '@bufferapp/ui/Loader';
+
+/** Loader Example */
+export default function ExampleLoader() {
+  return (
+    <Loader>
+      Hello, world!
```
Comment (federicoweber) [inline src/documentation/examples/Loader/Loader.jsx]: I don't think we are using the children text 😄

File: src/components/index.js [hunk 2]
```diff
@@ -11,4 +11,4 @@ export { default as SidebarHeader } from './SidebarHeader';
 export { default as Tooltip } from './Tooltip';
 export { default as Input } from './Input';
 export { default as ProgressBar} from './ProgressBar';
-export { default as SocialButton} from './SocialButton';
\ No newline at end of file
+export { default as SocialButton} from './SocialButton';export { default as Loader } from './Loader';
```
Comment (federicoweber) [inline src/components/index.js]: Might be a bit cleaner on a separate line 😄.

File: src/documentation/examples/Loader/Loader.jsx (hunk 1 above)
Comment (gmzjuliana) [reply src/documentation/examples/Loader/Loader.jsx]: good catch!

File: src/components/index.js (hunk 2 above)
Comment (gmzjuliana) [reply src/components/index.js]: yes haha , thanks fede

**Comment by gmzjuliana** [comment]
> Hey @gmzjuliana I left a few small comments, and I have a question for you, but overall this is looking great 🥇.
> 
> I noticed you are animating using solid colors, including white: I was under the impression that we wanted to animate with opacity and have the loader elements completely disappear on the white phase.
> But I'm not sure if I'm missing any context in here 😅.

heya @federicoweber ,  I hadn't thought of that but is that so we can use the loader on different coloured backgrounds? I can definitely change that in that case 😊 otherwise I built it this way because the figma file had solid colours 🙈 [here's the file for reference](https://www.figma.com/file/onxcG6l5KavWZ4Z6gt8k9M/Loader-Component-Animation?node-id=0%3A1)

**Comment by federicoweber** [comment]
Hey @gmzjuliana, yep that is mainly for the background, as it seems a bit off on gray. Might be worth checking with design on that 😄.

**Comment by gmzjuliana** [comment]
heya @federicoweber had some time to come clean this up :) I changed the white to transparent but left the solid colours in there, how does that feel?

**Pull request merged**